Sources/SwiftDiagnostics/FixIt.swift

```diff
@@ -46,3 +46,48 @@ public struct FixIt {
     self.changes = changes
   }
 }
+
+public extension FixIt {
+  var edits: [SourceEdit] {
+    var existingEdits = [SourceEdit]()
+    for change in changes {
+      let edit = edit(from: change)
+      var isOverlapping = false
+      for existingEdit in existingEdits {
+        // The edit overlaps with the previous edit. We can't apply both
+        // without conflicts. Apply the one that's listed first and drop the
+        // later edit.
+        if edit.range.overlaps(existingEdit.range) {
+          isOverlapping = true
+          break
+        }
+      }
+      if !isOverlapping {
+        existingEdits.append(edit)
+      }
```

Comment:
I think you should be able to write this as 

```suggestion
      let isOverlapping = existingEdits.contains { edit.range.overlaps($0.range) }
      if !isOverlapping {
        // The edit overlaps with the previous edit. We can't apply both
        // without conflicts. Apply the one that's listed first and drop the
        // later edit.
        existingEdits.append(edit)
      }
```